libass/ass_render.c

```diff
@@ -553,17 +558,17 @@ static bool quantize_transform(double m[3][3], ASS_Vector *pos,
         }
 
     // x_lim = |m_xx| * dx + |m_xy| * dy
-    //  ~= |qm_xx| * q_x * dx + |qm_xy| * q_y * dy
+    //  ≈ |qm_xx| * q_x * dx + |qm_xy| * q_y * dy
```

Comment:
If we permit unicode in source code then `<=` and `^2` should be replaced with `≤` and `²`, also better to use other symbols (`×`, `±`, `∝`, `≡`, `⋅`, `Δ`, `mᵢⱼ`, etc.) too.